spec/integration/lib/auth0/api/v2/api_jobs_spec.rb

```diff
@@ -44,6 +44,29 @@
       end
     end
 
+    describe '.export_users and .get_job' do
+      let(:connection_id) do
+        client.connections
+              .find do |connection|
+          connection['name'].include?(Auth0::Api::AuthenticationEndpoints::UP_AUTH)
+        end['id']
+      end
+      let(:exported_users) { client.export_users }
+      it do
+        expect(exported_users).to include(
+          'connection' => Auth0::Api::AuthenticationEndpoints::UP_AUTH,
```

Comment:
... which is why this test will fail if you move it out of the `skip` block and run it.

Reply:
Sorry I don't really follow. This test seems to use the 'spec_helper' which does not include the Credentials module so I actually get the following error.

```


NameError:
  undefined local variable or method `v2_creds' for #<RSpec::ExampleGroups::Auth0ApiV2Jobs after(:context) hook>
# ./spec/integration/lib/auth0/api/v2/api_jobs_spec.rb:89:in `block (2 levels) in <top (required)>'
 1/6 |=== 16 ====>                                                            |  ETA: 00:00:00
Finished in 0.03837 seconds (files took 0.51953 seconds to load)
1 example, 1 failure, 1 error occurred outside of examples

Failed examples:

rspec ./spec/integration/lib/auth0/api/v2/api_jobs_spec.rb:32 
```

Reply:
If I use MODE=full I get the following error for all tests in this file, not just the one I added.
```
1) Auth0::Api::V2::Jobs.import_users and .get_job
     Failure/Error: raise InvalidApiNamespace, 'API namespace must supply an API domain' if @domain.to_s.empty?

     Auth0::InvalidApiNamespace:
       API namespace must supply an API domain
```

Reply:
@raldred - Thanks for sticking with this!

We've made a few changes to the testing suite that should make this work a bit more smoothly. On your local copy, rebase with our `master` and run:

```bash
$ bundle exec rake test
```

I just ran that process on your branch locally and everything passes in it's current state (assuming all of your changes are pushed). I moved your tests out of the skip block and they complete just fine in the state they're in. 

One thing to note ... you'll need to uncomment [this line](https://github.com/auth0/ruby-auth0/blob/master/spec/spec_helper.rb#L25) to allow HTTP connections temporarily when you move the tests out of the skip block. Otherwise VCR (records the HTTP calls so the tests ran faster and don't hammer our servers) will stop the call. 

Try the above and, if it works, just add the test descriptions and we're good to go. If not, ditch the integration tests and I'll handle them on my end. This file needs work generally and we need a CONTRIBUTING.md file to explain all of this stuff. 

Thanks again!